scripts/build-downstream-projects.sh

```diff
@@ -57,6 +58,8 @@ spl() {
     set -x
     rm -rf spl
     git clone https://github.com/solana-labs/solana-program-library.git spl
+    # copy toolchain file to use solana's rust version
+    cp "$solana_dir"/rust-toolchain.toml spl/
```

Comment:
it [breaks](https://buildkite.com/solana-labs/solana/builds/86746#01851b7c-20d9-4fa8-a926-8841aa5cee36) now while running `./ci/test-stable-perf.sh`, which is more complicated since I cannot reproduce locally without having cuda shared libraries :-(

Reply:
No luck :sweat: I have some local perf libs built, so I can take care of that script if you want